arbitrum-docs/launch-orbit-chain/how-tos/customize-arbos.mdx

```diff
@@ -0,0 +1,138 @@
+---
+title: "How to customize ArbOS on your Orbit chain"
+description: "Learn how to customize your ArbOS version"
+author: jasonwan
+sidebar_position: 7
+content_type: how-to
+---
+
+import PublicPreviewBannerPartial from '../partials/_orbit-public-preview-banner-partial.md';
+
+<PublicPreviewBannerPartial />
+
```

Comment:
```suggestion
```caution Customizations may not be for everyone
Being able to customize and control every aspect of your chain is one of the key benefits of building with the Arbitrum Orbit tech stack. Performing these customizations, however, is complex and may not be tenable for every team (depending on the type of changes your team is trying to make). 

Due to the complex nature of ArbOS customizations, we strongly recommend being very intentional with your chain's customizations and budget for extra resources to perform the short term work, but also to manage the medium and long term maintenance and security risks. Despite our desire to help every team with their customizations, the Offchain Labs team has limited capacity to review and audit every team's customizations but we will do our best, alongside your RaaS teams, to support you in any way we can.
```
```